src/utilcode/debug.cpp

```diff
@@ -349,13 +349,6 @@ bool _DbgBreakCheck(
     STATIC_CONTRACT_FORBID_FAULT;
     STATIC_CONTRACT_DEBUG_ONLY;
 
-    RaiseExceptionOnAssert(rTestAndRaise);
-
-    if (DebugBreakOnAssert())
-    {
-        DebugBreak();
-    }
-
```

Comment:
This does did fix it.  It was being checked in too many places so we would hit the DebugBreak about 3 or 4 times for every assert. Also 3 or 4 exceptions would be thrown if RaiseExceptionOnAssert was set.

I think it is going to be useful internally and to CSS if there are no symbols in libcoreclr (or even if they do exist) to stop after libcoreclr is loaded and initialized. Otherwise it is really hard to set a managed bp.